### ISSUE TITLE
Reproduction result

### ISSUE THREAD

**Issue by DongyangHuLi** [zzangjinsun/NLSPN_ECCV20#56]
Hi~
I used the pre-trained model you provided to visualize the results on the KITTI 1K selected validation set：
![image](https://user-images.githubusercontent.com/116615779/233317426-80f1c9e6-8f31-445c-92f0-02e3f2b83c75.png)
There seems to be a lot of black spots on the depth map. Is this normal?
Thank you.

**Comment by DongyangHuLi** [comment]
It has been solved.

**Comment by saythe17** [comment]
May I know how it was resolved? I had the same problem.
